apps/meteor/ee/app/livechat-enterprise/server/lib/VisitorInactivityMonitor.js

```diff
@@ -98,7 +98,7 @@ export class VisitorInactivityMonitor {
 		if (!action || action === 'none') {
 			return;
 		}
-		LivechatRooms.findAbandonedOpenRooms(new Date()).forEach((room) => {
+		LivechatRooms.findAbandonedOpenRooms(new Date().toISOString()).forEach((room) => {
```

Comment:
```suggestion
		LivechatRooms.findAbandonedOpenRooms(new Date()).forEach((room) => {
```

I don't think this is required since you're converting to string over here & then in `.findAbandonedOpenRooms` method you're converting it back to Date object. I already tested this in my local without string conversion and it seems to be working. Can you please check this part again. I feel we only need to change the way we watch the settings over here